src/sesg_cli/cli/pdf_to_txt.py

```diff
@@ -0,0 +1,54 @@
+import os
+from pathlib import Path
+
+import PyPDF2
+import typer
+from rich.progress import Progress
+
+app = typer.Typer(
+    rich_markup_mode="markdown", help="Convert pdfs to txt"
+)
+
+
+@app.command()
+def convert(
+    slr_folder_path: Path = typer.Argument(
+        ...,
+        help="Path to pdfs folder to be converted",
+        dir_okay=True,
+        exists=True,
+    )
+):
+    files: list[str] = os.listdir(f"{slr_folder_path}\\pdfs")
```

Comment:
this `os.listdir` statement wont work on linux due to the `\\`, use [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html#pathlib.Path) as it automatically handles path division

i think something like this would work:

```python
files = (slr_folder_path / "pdfs").iterdir()
```

files will be an iterable (not a list, but it doesnt make difference in this use case) of `pathlib.Path`

Reply:
Changed!!